CONTRIBUTING.md

```diff
@@ -206,28 +197,29 @@ There may be times that a test is already written - in these circumstances it is
+
+If you wish to add a new feature or you spot a bug that you wish to fix, **please open an issue for it first** in the [CORE project](https://vatsimuk.atlassian.net/projects/CORE/issues).
 
-If you wish to add a new feature or you spot a bug that you wish to fix, **please open an issue for it first** on the [UK Core Web Services Repository](https://github.com/vatsim-uk/core/issues).
+**Note:** You will need to signup to create issues within the project. See [Issue Tracking](ISSUE_TRACKING.md) for more information.
 
 The work-flow for submitting a new merge request is designed to be simple, but also ensure consistency from **all** contributors:
 
 * Fork the project into your personal space on GitHub.com
 * Create a new branch (with the name `<issue_number>-<name>`, replacing issue_number with the issue number you're resolving)
  * The exception to this is where an entire feature is being tackled, spanning multiple issues.  In this case you can create a `feature/<name>` branch.
 * Commit your changes
- * When writing commit messages, consider closing your issues via the commit message (by starting the message with "fix #22" or "fixes #22" and then your description.
-  * The issues will be referenced in the first instance and then closed once the MR is accepted.
-* **Add any important setps that must be followed on deployment to the README.md file**
+ * When writing commit messages, consider changing the state your issues via the commit message (by including the issue ID and the state name in your message `CORE-33 #testing #comment Fixed some stuff.`)
+* **Add any important steps that must be followed on deployment to the README.md file**
 * Push the commit(s) to your fork
-* Submit a merge request (MR) to **our** development branch
-* The MR title should describe the change that has been made
-* The MR description should confirm what changes have been made, how you know they're correct (with references)
+* Submit a Pull Request (PR) to **our** development branch
+* The PR title should describe the change that has been made and contain the issue ID. (`ISSUE_ID - SUMMARY`)
+* The PR description should confirm what changes have been made, how you know they're correct (with references)
  * It is expected that all submissions have associated test cases.
-* Ensure you link any relevant issues in the merge request (you can type hash and the issue ID, eg #275).  Comment on those issues linking back to the MR (you can reference MRs using the format !<MR_ID> for example !22).
-* Be prepared to answer any questions about your MR when it is reviewed for acceptance.
+* Ensure you link any relevant issues in the Pull Request by including the Issue ID in the title. This will automatically link the issue to the Pull Request.
```

Comment:
Does JIRA support linking these in the description, rather than the title?

Reply:
Unfortunately not. However, upon further reading, if the source branch of the Pull Request is linked to the issue, then the PR will automatically be linked. So I will update accordingly.